src/opera/test/util/test_metadata_utils.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2022, by the California Institute of Technology.
+# ALL RIGHTS RESERVED.
+# United States Government sponsorship acknowledged.
+# Any commercial use must be negotiated with the Office of Technology Transfer
+# at the California Institute of Technology.
+# This software may be subject to U.S. export control laws and regulations.
+# By accepting this document, the user agrees to comply with all applicable
+# U.S. export laws and regulations. User has the responsibility to obtain
+# export licenses, or other export authority as may be required, before
+# exporting such information to foreign countries or providing access to
+# foreign persons.
+#
+
+"""
+=================
+test_metadata_utils.py
+=================
+
+Unit tests for the util/metadata_utils.py module.
+
+"""
+
+import unittest
+from unittest import skipIf
+
+def mgrs_osr_gdal_are_available():
+    """
+    Helper function to check for a local installation of the Python bindings for
+    the Geospatial Data Abstraction Library (GDAL), osr and mgrs.
+    Used to skip tests that require GDAL if it's not available.
+    """
+    try:
+        from osgeo import gdal  # noqa: F401
+        from osgeo import osr  # noqa: F401
+        import mgrs  # noqa: F401
+        return True
+    except (ImportError, ModuleNotFoundError):
+        return False
+
+
+class MetadataUtilsTestCase(unittest.TestCase):
+    """Unit test Metadata Utilities"""
+
+    @skipIf(not mgrs_osr_gdal_are_available(), reason="gdal, osr, and/or mgrs is not installed on the local instance")
+    def test_get_geographic_boundaries_from_mgrs_tile(self):
+        from opera.util.metadata_utils import get_geographic_boundaries_from_mgrs_tile
```

Comment:
Please co-locate this with the rest of the imports at the top of the module

Reply:
I think I need more input on this.  I moved this into the @skipIf scope because otherwise it will raise an exception in get_geographic_boundaries_from_mgrs_tile() when it tries to import osgeo if it doesn't exist.  I don't think that we want to suppress the import exception in get_geographic_boundaries_from_mgrs_tile().

One thought is that I can add try/except to the import in test_metadata_utils.py:

try:
    # If mgrs, osr, or gdal are not available for import, this will raise an exception
    from opera.util.metadata_utils import get_geographic_boundaries_from_mgrs_tile
except:
    pass

Am I missing a more elegant solution for this?

Here is the exception raised if moved out of the skipIf scope:
```
Traceback:
../../../.conda/envs/opera/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
src/opera/test/util/test_metadata_utils.py:15: in <module>
    from opera.util.metadata_utils import get_geographic_boundaries_from_mgrs_tile
src/opera/util/metadata_utils.py:13: in <module>
    from osgeo import gdal
E   ModuleNotFoundError: No module named 'osgeo'
```